src/commands/Suggestion Commands/AcceptCommand.ts

```diff
@@ -64,6 +64,18 @@ export const run: RunFunction = async (client, message, args) => {
 			color: 'GREEN',
 		})
 	);
+	
+const author: User = client.users.cache.get((Suggestion as Anything).User);
+	if (author) {
+		author.send(
+			new MessageEmbed({
```

Comment:
Can we use the `client.embed` function rather then constructing manually? It'd show the user that accepted it.

Reply:
Implementing now.

Reply:
FYI: Usage is `client.embed(stuff inside messageembed, message)`

Reply:
i am such. an. idiot.

Reply:
Don't worry about it. Seriously!